kernelci/test.py

```diff
@@ -81,6 +81,7 @@ def get_params(bmeta, target, plan_config, storage):
     url_px = file_server_resource
     job_name = '-'.join([job_px, dtb or 'no-dtb',
                          target.name, plan_config.name])
+    job_filename = '-'.join([job_px, target.name, plan_config.name])[:200]
```

Comment:
The issue is not the length of the file name, but the length of the LAVA job name.

Reply:
Right, I misunderstood it. Corrected.